test/e2e/tests/settings/account-token-list.spec.js

```diff
@@ -56,11 +56,23 @@ describe('Settings', function () {
           text: 'General',
           tag: 'div',
         });
-        await driver.clickElement({ text: 'Fiat', tag: 'label' });
+        await driver.clickElement('.show-native-token-as-main-balance');
+        // We now need to enable "Show fiat on testnet" if we are using testnets (and since our custom
+        // network during test is using a testnet chain ID, it will be considered as a test network)
+        await driver.clickElement({
+          text: 'Advanced',
+          tag: 'div',
+        });
+        await driver.clickElement('.show-fiat-on-testnets-toggle');
+        // Looks like when enabling the "Show fiat on testnet" it takes some time to re-update the
+        // overview screen, so just wait a bit here:
+        await driver.delay(1000);
```

Comment:
I have had to do this a couple times in my PR too. Wondering if you know of any other ways to implement this sort of thing without having to `delay`?

Reply:
Hmmmm not sure tbh, it works in most cases without it, but sometimes to avoid the flakiness we add the small delay